agent/php_user_instrument.c

```diff
@@ -273,6 +261,8 @@ static void nr_php_user_wraprec_destroy(nruserfn_t** wraprec_ptr) {
     return;
   }
 
+  wraprec_id_dtor(&wraprec->id);
```

Comment:
Maybe a good time to change the '0's on line 256 and 260 to NULLs?

Reply:
Although it's a good suggestion, I'd rather keep the PR focused and not have it change the code it doesn't have to. I'd say the `0` -> `NULL` style change in 256 and 260 is not related to this PR and IMHO deserves its own PR.

Reply:
True but like some other similar chores (PHP7 defines, etc) we've been doing it opportunistically as we update nearby code. 
 Just a consideration.